docs/source/development/godocs.rst

```diff
@@ -0,0 +1,57 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+******
+Godocs
+******
+
+Linking to Godocs
+=================
+
+As mentioned in :ref:`docs-guide`, you can use the ``:docs-godoc:`` role, the ``:docs-atc-godoc:`` role, and the ``:docs-to-godoc:`` role to link to Godocs from the ATC documentation.
```

Comment:
there's no `docs-` prefix on those roles

Reply:
Right you are, `docs-` prefix removed in 1b8cd21b82.